examples/specs_integration/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "specs_integration"
+version = "0.1.0"
+authors = ["tom.leys <tom@crump-leys.com>", "karroffel <therzog@mail.de>"]
```

Comment:
Maybe this should be "The godot-rust developers" (or whatever the proper wording is) that all other crates use too?